tests/v2/validation/snapshot/snapshot.go

```diff
@@ -49,6 +50,13 @@ const (
 	WorkloadNamePostBackup       = "wload-after-backup"
 )
 
+type snapshotStruct struct {
+	kubernetesVersion string
+	controlPlaneValue string
```

Comment:
nit; sorry to bring this up again but I don't feel the name here matches what the variable will be used for. Could you replace `value` with something like `maxUnavailable` or something to show which part of the snapshot these values are used for?

Reply:
The issue is that v2 prov and RKE1 use different naming conventions, so I just went with value to not have to create new parameters.

Reply:
I'll just have the name be initialValue in both cases so it's more clear.